src/posit/connect/bundles.py

```diff
@@ -235,16 +235,16 @@ def __init__(
         super().__init__(params)
         self.content_guid = content_guid
 
-    def create(self, input: io.BufferedReader | bytes | str) -> Bundle:
+    def create(self, archive: io.BufferedReader | bytes | str) -> Bundle:
```

Comment:
Is this to avoid using `input` as an argument?

Reply:
Correct. Since `input` is a builtin function.

Reply:
We might not need to worry cause this is internal enough or no one is yet using it, but does this change have user-facing implications?

Reply:
Yes, it does if anyone is using kwargs when calling this method. Since it's early and we are pre-1.0.0, I am ok releasing this under 0.4.0 along with documentation of the breaking changes.

Reply:
Yes, I think release notes are the correct place to write them.

Reply:
I started a migration guide in the changelog: https://github.com/posit-dev/posit-sdk-py/releases